backend/src/Service/ChatRoom/ChatRoomService.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Service\ChatRoom;
+
+use App\AutoMapping;
+use App\Entity\ChatRoomEntity;
+use App\Manager\ChatRoom\ChatRoomManager;
+use App\Response\ChatRoom\ChatRoomResponse;
+use App\Response\ChatRoom\ChatRoomsStoreResponse;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+class ChatRoomService
+{
+    private AutoMapping $autoMapping;
+    private ChatRoomManager $chatRoomManager;
+    private string $params;
+
+    public function __construct(AutoMapping $autoMapping, ChatRoomManager $chatRoomManager, ParameterBagInterface $params)
+    {
+        $this->params = $params->get('upload_base_url') . '/';
```

Comment:
There is UploadFileHelperService that we have to use instead of this way of getting upload base url.

Reply:
The entity related to the images was not merged so I took this action to run the front matters